### PR TITLE
Issue #12207: Documentation fixed for the Debug Section

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #12207

### CODE REVIEW

**Comment by rnveach** [comment]
GitHub, generate website

**Comment by NitishKumar06** [comment]
@rnveach @romani Any changes from your side required? else I believe that this PR is ready to be merged.

**Review by romani — APPROVED** [review]

File: src/xdocs/idea.xml [hunk 1]
```diff
@@ -54,6 +54,11 @@
           <img alt="screenshot" src="images/gui_screenshot_debug_idea.png"/>
         </span>
         <br/><br/>
+        Open the Test's source file by double-click on it in a source tree as is shown:<br/><br/>
+        <span class="wrapper block">
+          <img alt="screenshot" src="images/gui_screenshot_click_Test_File.png"/>
```
Comment (rnveach) [inline src/xdocs/idea.xml:59]: The new image looks very skewed compared to the others. It's like it was resized but not kept proportional to it's original. Compared to other images, left hand side looks a bit blurry and hard to read.

Can we please fix this?

**Review by rnveach — CHANGES_REQUESTED** [review]
One issue only.

File: src/xdocs/idea.xml (hunk 1 above)
Comment (NitishKumar06) [reply src/xdocs/idea.xml:59]: Sure I'd do that.

Comment (NitishKumar06) [reply src/xdocs/idea.xml:59]: Done!

**Comment by NitishKumar06** [comment]
GitHub, generate website

File: src/xdocs/idea.xml (hunk 1 above)
Comment (rnveach) [reply src/xdocs/idea.xml:59]: Please leave resolving items to reviewers.

**Comment by romani** [comment]
GitHub, generate website

**Comment by romani** [comment]
Image is still bad, but better than no image

**Pull request merged**